src/main/java/com/sprint/findex_team6/controller/IndexValController.java

```diff
@@ -0,0 +1,27 @@
+package com.sprint.findex_team6.controller;
+
+import com.sprint.findex_team6.dto.dashboard.IndexChartDto;
+import com.sprint.findex_team6.entity.PeriodType;
+import com.sprint.findex_team6.service.IndexValService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/index-val")
+public class IndexValController {
+  private final IndexValService indexValService;
+  public IndexValController(IndexValService indexValService) {
+    this.indexValService = indexValService;
+  }
```

Comment:
```suggestion
@RequiredArgsConstructor
public class IndexValController {
  private final IndexValService indexValService;
```
@RequiredArgsConstructor를 사용하는 건 어떤가요? 더 깔끔하게 코드를 작성할 수 있어요.